packages/fuel-indexer-macros/src/decoder.rs

```diff
@@ -833,7 +833,7 @@ impl From<ObjectDecoder> for TokenStream {
                                             .map(|query| query.to_string())
                                             .collect::<Vec<_>>();
 
-                                        d.lock().await.put_many_to_many_record(queries).await;
+                                        d.lock().await.put_many_to_many_record(queries).await.unwrap();
```

Comment:
- No `unwrap()` unless you can prove it'll never panic
- Let's fail with a helpful error message here